src/ChannelServer/Scripting/ScriptManager.cs

```diff
@@ -439,6 +439,8 @@ private int addnpc(IntPtr L)
 			if (map == null)
 				return Melua.melua_error(L, "Map '{0}' not found.", mapName);
 
+            if (name.StartsWith("ETC_") || name.StartsWith("QUEST_"))
```

Comment:
Please set your IDE to use tabs, as outlined in our coding conventions, which can be found in the [readme](https://github.com/aura-project/melia#coding-conventions).